systemd/ceph-exporter@.service.in

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=Ceph cluster exporter daemon
+PartOf=ceph-exporter.target
+After=network-online.target local-fs.target time-sync.target
```

Comment:
why `time-sync`?

Reply:
This is to prevent a clock skew. `time-sycn.target` is first introduced by [this issue](https://tracker.ceph.com/issues/15419). According to the issue, ceph-{mds,mon,osd,radosgw} should be started after ntp (or systemd-timesyncd or chrony) to avoid the clock skew.

Because ceph-exporter also has some logic dependent on the clock, I consider this is also requred for ceph-exporter.
ref. https://github.com/cybozu/ceph/blob/60a15fc77fc2c1e0af530118a5513dc37c34b10e/src/exporter/util.cc#L18

Reply:
we should have used the `steady_clock` for this purpose. right, but the reason was the mon warns at seeing clock skew. but this does not apply to ceph-exporter, which does not exchange or check the timestamp with other nodes in the cluster -- it just exports metrics. so i still think it does not need to wait for the synchronization with ntpd.

Reply:
> we should have used the `steady_clock` for this purpose.

implemented in #55559

Reply:
I removed `time-sync.target` from both `After` and `Wants` settings.